sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/PlanParserSuite.scala

```diff
@@ -1677,25 +1677,24 @@ class PlanParserSuite extends AnalysisTest {
         ScriptInputOutputSchema(List.empty, List.empty, None, None,
           List.empty, List.empty, None, None, false)))
 
-    // verify with ROW FORMAT DELIMETED
-    assertEqual(
-      """
-        |SELECT TRANSFORM(a, b, c)
-        |  ROW FORMAT DELIMITED
-        |  FIELDS TERMINATED BY '\t'
-        |  COLLECTION ITEMS TERMINATED BY '\u0002'
-        |  MAP KEYS TERMINATED BY '\u0003'
-        |  LINES TERMINATED BY '\n'
-        |  NULL DEFINED AS 'null'
-        |  USING 'cat' AS (a, b, c)
-        |  ROW FORMAT DELIMITED
-        |  FIELDS TERMINATED BY '\t'
-        |  COLLECTION ITEMS TERMINATED BY '\u0004'
-        |  MAP KEYS TERMINATED BY '\u0005'
-        |  LINES TERMINATED BY '\n'
-        |  NULL DEFINED AS 'NULL'
-        |FROM testData
-      """.stripMargin,
+    // verify with ROW FORMAT DELIMITED
+    val sqlWithRowFormatDelimited =
+        "SELECT TRANSFORM(a, b, c)" + "\n" +
+        "  ROW FORMAT DELIMITED" + "\n" +
+      """  FIELDS TERMINATED BY '\t'""" + "\n" +
```

Comment:
friendly ping @AngersZhuuuu , do you have any suggestions about this `deprecated usage` issue